src/utils/CortexLookup.js

```diff
@@ -38,6 +38,7 @@ const navigationFormZoomArray = [
   'element:price',
   'element:rate',
   'element:code',
+  'parent',
```

Comment:
The 'load more' problem can also be resolved by adding the following zooms to the `navigationFormZoomArray`:

```
'element:items',
  'element:items:element',
  'element:items:element:availability',
  'element:items:element:definition',
  'element:items:element:definition:assets:element',
  'element:items:element:price',
  'element:items:element:rate',
  'element:items:element:code',
```